README.md

```diff
@@ -48,6 +58,7 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
     <input type="submit" value="Submit" id="submit">
   </form>
 ```
+var $(user_input) = $form.find(["input[id='color']").val();
```

Comment:
I think your way would work, but you can do it much simpler with `var $user_input = $('#color').val()`.